app/jobs/regular/create_chat_mention_notifications.rb

```diff
@@ -48,7 +48,7 @@ def send_os_notifications(membership)
                                  username: @creator.username
                                 ),
         tag: DiscourseChat::ChatNotifier.push_notification_tag(:mention, @chat_channel.id),
-        excerpt: @chat_message.excerpt,
+        excerpt: @chat_message.message[0..399],
```

Comment:
I know this is a revert, but why 399?

Reply:
Same question 🤔 That number seems quite magical

Reply:
don't question the magic

Reply:
IDK why I had this before 🤔

Reply:
I think maybe defining why the number or what it means is nice.

Separately, would also be nice if `send_notifications` could be extracted to test?

Reply:
There are some integration tests for chat message creation that run jobs immediately and test that push notifications are sent 🤷

Reply:
That's great!